### DISCUSSION TITLE
Add Pulumi Support

### DISCUSSION THREAD

**Discussion by atrauzzi** [aquasecurity/trivy#5413] [Ideas]
### Description

This one might be a little tricky, but I'd love to be able to run trivy against my pulumi codebase.

I author my pulumi templates in C#, so I think it'd be important to see if there's a way to consume whatever internal representation they use.

### Target

None

### Scanner

Misconfiguration

**Comment by AnaisUrlichs** [comment]
hi there, just came across this idea -- there is actually an integration by the pulumi community with Aqua enterprise https://github.com/pulumiverse/pulumi-aquasec

Any integration directly with Trivy would have to come from the community tho, would love to know if there are more people interested in that :)

**Reply by itaysk** [reply]
I think what the issue is about Trivy scanning Pulumi IaC and find issues there, similarly to how it finds issues with Terraform files. While the pulumi-aquasec extension seem to be about using Pulumi to manage Aqua.
